services/QuillLMS/spec/services/vitally_integration/serialize_vitally_sales_account_spec.rb

```diff
@@ -17,6 +17,17 @@
     )
   end
 
+  before do
+    previous_year_data = {
+      total_students: 2,
+      active_students: 1,
+      activities_finished: 1,
+      activities_per_student: 1.0
+    }
+    year = Date.today.year - 1
+    $redis.set("school_id:#{school.id}_vitally_stats_for_year_#{year}", previous_year_data.to_json)
```

Comment:
I think you can utilize your new cache object: `cached_data = CacheVitallySchoolData.set(school.id, year, previous_year_data)`